codegen/src/test/resources/software/amazon/awssdk/codegen/poet/waiters/query-sync-waiter-class.java

```diff
@@ -84,6 +89,15 @@ public void close() {
         return new DefaultBuilder();
     }
 
+    private <T extends QueryRequest> T applyWaitersUserAgent(T request) {
+        Consumer<AwsRequestOverrideConfiguration.Builder> userAgentApplier = b -> b.addApiName(ApiName.builder()
+                                                                                                      .version(VersionInfo.SDK_VERSION).name("WAITERS").build());
```

Comment:
In metric publisher we do

```
ApiName.builder().name("hll").version("cw-mp").build();
```

With the mapper we do 

```
ApiName.builder().version("ddb-enh").name("hll").build();
```

Waiters is kind of a high-level library. Should we do 

```
ApiName.builder().name("hll").version("waiter").build();
```

?

Reply:
Yea sure, I can change that. I was imitating the `Paginator` user agent so it looked like this.